docs/modules/ROOT/pages/guide-keep3r.adoc

```diff
@@ -47,37 +49,54 @@ image::guide-keep3r-relayer-balance.png[Checking the balance of your Relayer]
 
 NOTE: Defender will send you an email when your Relay's funds drop below 0.1 ETH to remind you to refill it.
 
-## Create an Autotask
+If you want to *bond* any KP3R tokens to your Relayer, you will need to transfer the tokens to the Relayer address as well. Make sure to send the tokens to the Relayer address and not to the KP3R token contract!
 
-Now that you have your Relay ready to send transactions, let's set up the Defender Autotask that will instruct the Relay to send them. An Autotask is a snippet of code that runs at regular intervals, in this case, we'll use it to automate the process of bonding, activating, searching for jobs, and executing them.
+## Bond your Relayer
```

Comment:
What about the opposite, unbond/deactivate a keeper?  We should include a line that this is/isn't supported via the UI and users will need to write a script to do this.  Ideally with an example of the script.

https://docs.keep3r.network/keepers#removing-a-keeper

Reply:
This is doable through the UI without resorting to scripts (although without "first class support" as it is for Bond),  using the "Send transaction" action from the Relayers settings page.

Reply:
Added!